test/remote/gateways/remote_epsilon_convenience_store_test.rb

```diff
@@ -16,6 +16,7 @@ def test_convenience_store_purchase_successful
       assert_match /\d{4}\-\d{2}\-\d{2} \d{2}:\d{2}:\d{2}/, response.params['receipt_date']
       assert_match /\d{4}\-\d{2}\-\d{2}/, response.params['convenience_store_limit_date']
       assert_match %r!\Ahttp://.+!, response.params['convenience_store_payment_slip_url']
+      assert_match /\d{5}/, response.params['receipt_number']
```

Comment:
receipt_number params already compared at Line 15.
https://github.com/pepabo/active_merchant-epsilon/blob/add-response-params/test/remote/gateways/remote_epsilon_convenience_store_test.rb#L15
 Is this correct?